apps/cookbook/src/app/examples/example-configuration-wrapper/example-configuration-wrapper.component.html

```diff
@@ -1,4 +1,15 @@
-<ng-container *ngIf="snapToViewport; else fieldset">
+<button
```

Comment:
For line 20: `<legend>Configuration</legend>` should be configurable via input prop but default to 'Configuration'